response_operations_ui/controllers/collection_instrument_controllers.py

```diff
@@ -15,3 +16,12 @@ def upload_collection_instrument(short_name, period, file):
     response = requests.post(url, files={"file": (file.filename, file.stream, file.mimetype)})
     if response.status_code != 201:
         raise ApiError(response)
+
+
+def get_collection_instruments(short_name, period):
+    logger.debug('Getting collection instruments', short_name=short_name, period=period)
+    url = f'{app.config["BACKSTAGE_API_URL"]}/collection-instrument/{short_name}/{period}'
+    response = requests.get(url)
+    if response.status_code != 200:
+        raise ApiError(response)
+    return json.loads(response.text)
```

Comment:
A debug level log line saying getting the collection instruments was successful here would be good for consistency with the other controller functions